tests/safeds/data/tabular/transformation/test_imputer.py

```diff
@@ -207,3 +207,52 @@ def test_should_not_change_original_table(self) -> None:
         )
 
         assert table == expected
+
+    def test_get_names_of_added_columns(self) -> None:
+        transformer = Imputer(strategy=Imputer.Strategy.Constant(1))
+        with pytest.warns(
+            UserWarning,
+            match="Imputer only changes data within columns, but does not add any columns.",
+        ), pytest.raises(TransformerNotFittedError):
+            transformer.get_names_of_added_columns()
+
+        table = Table(
+            {
+                "a": ["b"],
+            },
+        )
+        transformer = transformer.fit(table, None)
+        with pytest.warns(UserWarning, match="Imputer only changes data within columns, but does not add any columns."):
+            assert transformer.get_names_of_added_columns() == []
+
+    def test_get_names_of_changed_columns(self) -> None:
+        transformer = Imputer(strategy=Imputer.Strategy.Constant(1))
+        with pytest.raises(TransformerNotFittedError):
+            transformer.get_names_of_changed_columns()
+        table = Table(
+            {
+                "a": ["b"],
+            },
+        )
+        transformer = transformer.fit(table, None)
+        assert transformer.get_names_of_changed_columns() == ["a"]
+
+    def test_get_names_of_removed_columns(self) -> None:
+        transformer = Imputer(strategy=Imputer.Strategy.Constant(1))
+        with pytest.warns(
+            UserWarning,
+            match="Imputer only changes data within columns, but does not remove any columns.",
+        ), pytest.raises(TransformerNotFittedError):
+            transformer.get_names_of_removed_columns()
+
+        table = Table(
+            {
+                "a": ["b"],
+            },
+        )
+        transformer = transformer.fit(table, None)
+        with pytest.warns(
+            UserWarning,
+            match="Imputer only changes data within columns, but does not remove any columns.",
+        ):
+            assert transformer.get_names_of_removed_columns() == []
```

Comment:
Imputer doesn't do anything if there are no missing values

Reply:
You should use numerical Values for testing of the imputer.

Reply:
Changed Imputer testcases to numerical values.  
Added columns without missing values to the testcase.